src/Sylius/Behat/spec/Context/Domain/ProductContextSpec.php

```diff
@@ -45,7 +55,7 @@ function it_implements_context_interface()
     }
 
     function it_deletes_a_product_variant_from_the_repository(
-        ProductVariantRepositoryInterface $productVariantRepository,
+        VariantRepositoryInterface $productVariantRepository,
```

Comment:
This should be reverted since we inject `ProductVariantRepositoryInterface $productVariantRepository` into the constructor, so we should operate on this interface everywhere else also.